src/kvstore/RocksEngine.h

```diff
@@ -52,6 +52,40 @@ class RocksRangeIter : public KVIterator {
     rocksdb::Slice end_;
 };
 
+class RocksRangeWithPrefixIter : public KVIterator {
```

Comment:
Why we need it?

Reply:
It seems RangeIter is enough for your case.

Reply:
No, because we also need to keep a prefix of partId.
This is RangeIter valid function, if we use this, we will scan data of other partition.
```
bool valid() const override {
    return !!iter_ && iter_->Valid() && (iter_->key().compare(end_) < 0);
}
```

Reply:
Make sense. Could we extend prefixIterator for it.  Not reimplement a new one.

Reply:
Good suggestion